targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F413xH/CMakeLists.txt

```diff
@@ -2,11 +2,21 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if("DISCO_F413ZH" IN_LIST MBED_TARGET_LABELS)
-    add_subdirectory(TARGET_DISCO_F413ZH)
+    target_include_directories(mbed-core INTERFACE TARGET_DISCO_F413ZH)
+    set(PERIPHERALPINS_FILE TARGET_DISCO_F413ZH/PeripheralPins.c)
+    set(SYSTEM_CLOCK_FILE TARGET_DISCO_F413ZH/system_clock.c)
 elseif("MTS_DRAGONFLY_F413RH" IN_LIST MBED_TARGET_LABELS)
-    add_subdirectory(TARGET_MTS_DRAGONFLY_F413RH)
+    target_include_directories(mbed-core INTERFACE TARGET_MTS_DRAGONFLY_F413RH)
+    set(PERIPHERALPINS_FILE TARGET_MTS_DRAGONFLY_F413RH/PeripheralPins.c)
+    set(SYSTEM_CLOCK_FILE TARGET_MTS_DRAGONFLY_F413RH/system_clock.c)
+
+# TODO CMAKE: Should the ONBOARD_TELIT_HE910.cpp and ONBOARD_TELIT_HE910.h
+# below be moved to mbed-os/connectivity/drivers/cellular/TELIT/HE910 ?
+    target_sources(mbed-core INTERFACE ONBOARD_TELIT_HE910.cpp)
```

Comment:
@jeromecoutant @0xc0170 should this file be moved where suggested?

Reply:
I would vote yes